src/errors/definitions.js

```diff
@@ -4,55 +4,262 @@
  * @enum { string }
  */
 const ErrorCodes = {
+
   // IDV
+
+  /**
+   * Reason: Missing required property when sending UCAs
+   * Troubleshooting: Make sure the provided UCA has all the required properties filled
+   */
   ERROR_IDV_UCA_MISSING_PROPERTY: 'error.idv.uca.missing.property',
+
+  /**
+   * Reason: UCA has no more retries remaining
+   * Troubleshooting: The maximum number of retries has reached.
+   * You must request a new credentialRequest and start over again
+   */
   ERROR_IDV_UCA_NO_RETRIES: 'error.idv.uca.no.retries',
+
+
+  /**
+   * Reason: The process is in a final status "FAILED","COMPLETED"
+   * Troubleshooting: if you got a FINAL state error,
+   * you must request a new credentialRequest and start over again
+   */
   ERROR_IDV_PROCESS_HAS_FINAL_STATUS: 'error.idv.process.has.final.status',
+
+  /**
+   * Reason: The UCA is in a final status
+   * Troubleshooting: Make sure to change the state of a UCA that is not in the final status.
+   * you must request a new credentialRequest and start over again
+   */
   ERROR_IDV_UCA_HAS_FINAL_STATUS: 'error.idv.uca.has.final.status',
+
+  /**
+   * Reason: A UCA of the batch is in a final status
+   * Troubleshooting: Make sure to change the state of a UCA that is not in the final status.
+   */
   ERROR_IDV_UCA_BATCH_HAS_FINAL_STATUS: 'error.idv.uca.batch.has.final.status',
+
+  /**
+   * Reason: The credentialItem is not valid/unknown to the IDV
+   * Troubleshooting: Make sure to provide the valid identifier of a credentialItem
+   */
   ERROR_IDV_CR_INVALID_CREDENTIAL_ITEM: 'error.idv.cr.invalid.credentialItem',
+
+  /**
+   * Reason: The signature could not be verified
+   * Troubleshooting: Try to sign again a credential
+   */
   ERROR_IDV_CREDENTIAL_INVALID_SIGNATURE: 'error.idv.credential.invalid.signature',
+
+  /**
+   * Reason: The credential has already been signed by the user
+   * Troubleshooting: Don't try to sign again a credential
```

Comment:
it could be that a user in the mobile makes the 'sign' request twice by mistake, so it gets signed ok once, then the idv-toolkit receives another request after it's been signed